spec/proto/extension/v1/oss.proto

```diff
@@ -0,0 +1,1059 @@
+//The file defined base on s3 protocol, to get an in-depth walkthrough of this file, see:
+//https://docs.aws.amazon.com/s3/index.html
+//https://github.com/aws/aws-sdk-go-v2
+syntax = "proto3";
+
+package spec.proto.extension.v1;
+
+import "google/protobuf/empty.proto";
+
+option go_package = "mosn.io/layotto/spec/proto/extension/v1;s3";
+
+service ObjectStorageService{
+    //Init oss client
+    rpc InitClient(InitInput) returns (google.protobuf.Empty){}
```

Comment:
So this API aims to do some "lazy init"?
I prefer to add a generic interface to do this type of jobs, for every types of components.
Can we delete this method here or move it to some other places?

Reply:
I expect ObjectStorageService to be an independent service, and it is a complete protocol definition. Even users who are not layotto users can use this complete protocol, so it may be necessary to retain the init operation？